bad_connector_woocommerce/views/product_template_view.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Inherit product template view to add connector page and  Binding -->
+    <record id="product_template_form_view_woo_inherit" model="ir.ui.view">
+        <field name="name">product.template.form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_only_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//header" position="after">
```

Comment:
Implement the stock update functionality for the product template and merge the PR of  https://github.com/BizzAppDev-Systems/connector-woocommerce/pull/51